library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx

```diff
@@ -30,6 +33,33 @@ export const ImageFileUploader: FileUploaderType = (
   const fileSize = fileValue?.file ? fileValue.file?.size : fileValue?.fileSize;
   const fileSizeMb = Math.round(((fileSize || 0) / 1024 / 1024) * 10) / 10;
 
+  const apiDownload = useStoreActions((actions) => {
+    return actions.api.download;
+  });
+  const [imageSrc, setImageSrc] = useState('' as string | null);
+
+  useEffect(() => {
+    try {
+      if (fileValue.file) {
+        const reader = new FileReader();
+        reader.onloadend = () => {
+            setImageSrc(reader.result as string);
+        };
+        reader.readAsDataURL(fileValue.file);
+      } else {
+        apiDownload({
+          path: downloadPath as string,
+          params: {},
+          successCallback: async (data: Record<string, any> | Array<Record<string, any>> | Blob) => {
+            setImageSrc(URL.createObjectURL(data as Blob))
+          },
+        });
+      }
+    } finally {
+      setImageSrc(null);
+    }
```

Comment:
```suggestion
     setImageSrc(null);

      if (fileValue.file) {
        const reader = new FileReader();
        reader.onloadend = () => {
            setImageSrc(reader.result as string);
        };
        reader.readAsDataURL(fileValue.file);
        
        return;
      }

       apiDownload({
          path: downloadPath as string,
          params: {},
          successCallback: async (data: Record<string, any> | Array<Record<string, any>> | Blob) => {
            setImageSrc(URL.createObjectURL(data as Blob))
          },
       });

```
Would be easier to read / understand